src/main/java/org/codehaus/groovy/tools/javac/JavaStubGenerator.java

```diff
@@ -709,6 +709,8 @@ private void printMethod(PrintWriter out, ClassNode clazz, MethodNode methodNode
         }
 
         printGenericsBounds(out, methodNode.getGenericsTypes());
+        if (methodNode.isDefault())
+            out.print(" default ");
```

Comment:
The tailing space could be removed, otherwise `assert stubSource.contains('default void method')` will fail, because `out.print(" ");` in the next line will add an additional space.